activemodel/lib/active_model/type/immutable_string.rb

```diff
@@ -7,10 +7,14 @@ def type
 
       def serialize(value)
         case value
-        when ::Numeric, ActiveSupport::Duration then value.to_s
-        when true then "t"
-        when false then "f"
-        else super
+        when ::Numeric, ActiveSupport::Duration then
+          value.to_s
+        when true then
+          "t"
+        when false then
+          "f"
+          else
```

Comment:
Yeah, this is those "wrong" linting rules that causes this. Let me try to fix this.